networking_cisco/tests/unit/cisco/device_manager/test_aci_vlan_trunking_driver.py

```diff
@@ -453,3 +452,56 @@ def _return_mocked_net(self, args):
                     self.plugging_driver.extend_hosting_port_info(ctx,
                         fake_port_db_obj, hosting_device, hosting_info)
                     self.assertEqual(hosting_info['segmentation_id'], 3003)
+
+    def test_allocate_hosting_port_info_adds_segment_id(self):
+        self.plugging_driver._default_ext_dict = {
+            'gateway_ip': '1.103.2.1',
```

Comment:
the naming of these parameters are still not consistent with the naming on the neutron side.